__tests__/index-test.js

```diff
@@ -0,0 +1,24 @@
+require("babel-polyfill");
+
+describe('Hacker News Plugin', () => {
+  it('execute returns new 10 items', async () => {
+    const hack = require('../index.js');
```

Comment:
Let's hoist these `require()`'s to the top, we don't need to `require()` for each spec.